gcp/rakefiles/utils.rb

```diff
@@ -0,0 +1,13 @@
+# Obfuscate sensitive data in the output of a command, relying on @secrets
+def sh_filter(*cmd)
+  if @secrets
+    IO.popen(*cmd).each do |out|
+      @secrets.each do |key, val|
+        out = out.gsub("#{val}", "<SENSITIVE>")
```

Comment:
I would do `out.gsub!` instead. This may be more performant than passing a modified copy back to the original variable? Anyway it's shorter :).